RSAKeyVaultProvider/RSAKeyVault.cs

```diff
@@ -134,5 +133,37 @@ public override byte[] EncryptValue(byte[] rgb)
         {
             throw new NotSupportedException();
         }
+#endif
+
+        private static HashAlgorithm GetHashAlgorithm(HashAlgorithmName algorithm)
+        {
+#if NETSTANDARD20
```

Comment:
I'm not sure it makes sense to have two separate code paths for NS2 and NS1. The if statements are functional enough.

Azure Key Vault doesn't support SHA1 for signing, so it may be worth letting it fall in to the NSE.